examples/sendgrid-event-webhook-consumer/Tests/EventWebhook.Tests/EventTests.cs

```diff
@@ -0,0 +1,376 @@
+using EventWebhook.Models;
+using EventWebhook.Parser;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace EventWebhook.Tests
+{
+    public class EventTests
+    {
+        [Fact]
+        public async Task AllEvents()
+        {
+            var jsonStream = new MemoryStream(File.ReadAllBytes("TestData/events.json"));
+            
+            IEnumerable<Event> events = await EventParser.ParseAsync(jsonStream);
+
+            events.Count().ShouldBe(11);
```

Comment:
Prefer to test only that there's at least 1 event. Makes it easier to add more events later w/o needing to keep this in-sync.

Reply:
Should I remove the model classes too?

Reply:
Ah.. got it. Kept the Processed event as example and removed other events

Reply:
Miscommunication here. I'm fine with all the other classes for various event types and think those should be put back, along with the other tests and sample data.

My comment was more about maintenance. Right now there are exactly 11 events in the sample event file. But later I may want to add another one to test out some edge-case parsing or whatever. I'd also have to go into this test file and update the number to 12.

When rather this test doesn't really care exactly _how many_ events were processed; just that they were all processed and there were no exceptions thrown. That's why I think it should just be testing that the deserialized events size is at least 1, since if any of them failed to process then an exception would have been raised.

Reply:
Bump in case this comment was missed.

Reply:
Thanks @childish-sambino, I'd missed that comment - reverted the commit and updated the test as per your suggestion.